met-web/src/components/engagement/view/EngagementView.tsx

```diff
@@ -82,9 +85,9 @@ export const EngagementView = () => {
                                     <SurveyBlock startSurvey={handleStartSurvey} />
                                 </Grid>
                             </Grid>
-                            <Grid item data-testid={'widget-block'} xs={12} lg={4}>
+                            {/* <Grid item data-testid={'widget-block'} xs={12} lg={4}>
```

Comment:
temporary change?

Reply:
This will disable all widgets in the engagement view right?

Reply:
This was experimental during development. I shouldn't have pushed it. Reverting. Thank you!